source/includes/usage-examples/sample-index-app-intro.rst

```diff
@@ -0,0 +1,11 @@
+You can use the following sample application to test the code examples on this
```

Comment:
This seems out of place, given other examples in the documentation don't provide such information.  Generally, they seem to provide a block about the examples using Project reactor.